quil-rs/src/program/graph.rs

```diff
@@ -587,9 +599,7 @@ impl<'a> ScheduledProgram<'a> {
                 | Instruction::Declaration(_)
                 | Instruction::GateDefinition(_)
                 | Instruction::FrameDefinition(_)
-                | Instruction::MeasureCalibrationDefinition(MeasureCalibrationDefinition {
-                    ..
```

Comment:
Any idea why this variant was written differently from all the others?

Reply:
No idea, probably just leftover from previous development.